package.json

```diff
@@ -39,14 +40,13 @@
     "template-helpers": "^1.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.3.4",
```

Comment:
it looks like we removed this dependency, is this intentional?

Reply:
This was intentional. I could not find a place it was used. We use `@babel/runtime-corejs3` in our asset creation process, but I couldn't see where `@babel/core` was used. It may be a hold-over from how we used to do things.